ApplicationPattern.yaml

```diff
@@ -4687,15 +4668,14 @@ paths:
                           type: string
                           description: >
                             'Label that shall be presented on the button
-                            from [/core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-001/action-profile-1-0:action-profile-pac/action-profile-capability/label or /core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-002/action-profile-1-0:action-profile-pac/action-profile-capability/label]'
+                            from [/core-model-1-4:control-construct/profile-collection/profile=*-1-0-0-action-p-*/action-profile-1-0:action-profile-pac/action-profile-capability/label'
```

Comment:
Same as above comment, a small typo in description of "label" parameter. A closing bracket ']' is missing at end

Reply:
Added the ']'. Thanks for indicating.